src/views/forms/TradeForm.tsx

```diff
@@ -186,6 +189,10 @@ export const TradeForm = ({
     alertType = inputAlert?.type;
   }
 
+  const isLiquidationError = ['MARKET_ORDER_ERROR_ORDERBOOK_SLIPPAGE'].some(
+    (errorCode) => inputAlert?.code === errorCode
```

Comment:
parsed through the abacus + localization code - this should be the only error we have to catch (other ones aren't in v4 i.e. `LIMIT_PRICE_TRIGGER_PRICE_SLIPPAGE_HIGHER`, `TAKE_PROFIT_MARKET_ORDER_MAY_NOT_EXECUTE`)